.github/hooks/pre-commit

```diff
@@ -1,3 +0,0 @@
-#!/bin/sh
-isort .
-flake8
```

Comment:
I don't think this anything runs this script, added in PR #331.

Reply:
Yeah that's not needed anymore. The docker build already takes care of that so we can remove it